secretmessage/secretmessage.go

```diff
@@ -2,5 +2,22 @@ package secretmessage
 
 // Decode func
 func Decode(encoded string) string {
-	return ""
+	// count
+	counts := make(map[rune]int)
+	for _, r := range encoded {
+		counts[r]++
+	}
+	// only get runes >= counts[_]
+	var runes []rune
+	for k, v := range counts {
+		if k != '_' && v >= counts['_'] {
+			runes = append(runes, k)
+		}
+	}
+	// sort runes by count
+	sort.Slice(runes, func(i, j int) bool {
+		return counts[runes[i]] > counts[runes[j]]
+	})
+	// return string
+	return string(runes)
```

Comment:
I really like this solution, it's better than the one i submitted. I come from a Java background so my solution looks clunky (and java like). I also didn't know that i can do sorting the way you did it :)

Reply:
Also as a meta comment, the commit adding the missing "sort" import should have been squashed in the initial commit, since all the commits originated from your side, and won't affect any upstream users, you could've just squashed the commits before the pull request

Reply:
You are correct, I should have squashed the second commit into the first.